src/overcooked_demo/server/config.json

```diff
@@ -5,13 +5,14 @@
     "MAX_GAME_LENGTH" : 120,
     "AGENT_DIR" : "./static/assets/agents",
     "MAX_FPS" : 30,
-    "psiturk" : {
+    "predefined" : {
         "experimentParams" : {
             "layouts" : ["counter_circuit", "cramped_room"],
-            "gameTime" : 11,
+            "gameTime" : 5,
```

Comment:
Is this 5 seconds? I'm guessing that's fine given that this file is just for demonstration purposes, right?

Reply:
Yeah it is 5 seconds. This was mostly so I could test the behaviors in less time during development. I don't know how it was designed to be used, but currently this only acts as a demo for how you can run predefined experiments